ts-packages/web/src/app/spaces/deliberation/[id]/components/space-contents.tsx

```
@@ -0,0 +1,95 @@
+}
+
+export default function SpaceContents({
+  isEdit = false,
+  htmlContents,
+  setContents = () => {},
+}: SpaceContentsProps) {
+  const html = (
+    <BorderSpaceCard>
+      <div
+        className="rich-content"
+        dangerouslySetInnerHTML={{ __html: htmlContents }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)